ui/src/Components/Dashboard/Table/Table.js

```diff
@@ -0,0 +1,68 @@
+import {useState,useEffect} from "react"
+
+
+import TableHeader from "./TableHeader/TableHeader"
+import TableRow from "./TableRow/TableRow"
+
+export default function Table({ featureFlags, headers }) {
+
+const [rowData,setRowData] = useState(featureFlags)
+
+useEffect(()=>{
+  console.log("called")
+})
+
+
+const handleSorting = (type) => {
+
+  if(type==="ascending")
+  {
+    let rows = rowData
+    rowData.sort((a,b)=>{
+      if(a.createdAt > b.createdAt) return 1
+
+      if(a.createdAt < b.createdAt) return -1
+
+      return 0;
+    })
+
+    setRowData([...rows]);
+  }
+
+  else if(type=="descending")
+  {
+    let rows = rowData
+    rows.sort((a,b)=>{
```

Comment:
This should be in `util` and we should be reusing the function instead of repeating the same code